Cargo.lock

```diff
@@ -1767,9 +1767,12 @@ dependencies = [
 
 [[package]]
 name = "num-traits"
-version = "0.2.5"
+version = "0.2.10"
```

Comment:
We don't need to update `num-traits` in this commit.

Reply:
@sgkim126 I don't know why `num-traits` was updated. It is probably because of the Cargo version; It's just my guess. But I think it's not a serious problem. Do you think this must be modified?

Reply:
A commit should contain changes only needed. If the changes are not relevant to your work, please remove it.